src/extension.ts

```diff
@@ -1,9 +1,40 @@
 import { ExtensionContext } from "vscode";
 import { GodotTools } from "./godot-tools";
-
+import * as path from "path";
+import * as fs from "fs"
+import * as vscode from 'vscode'
 
 let tools: GodotTools = null;
 
+export class DocContent {
```

Comment:
You'd better move this class in to a script file and instance it as a member of `GodotTools`.
In this way we can keep this entry script as simple as possiable.